app/templates/eventView.html

```diff
@@ -57,49 +57,25 @@ <h3><br>Description</h3>
                 </div>
                 <div class="col-md-6">
                     <div class="col-sm">
-                        <h3><br>Program Manager</h3>
-                      {% if programManager %}
-                      <p>
-                          {{programManager.user.firstName}} {{programManager.user.lastName}}<br>
-                          {{programManager.user.email}}
-                      </p>
-                      {% else %}
-                      <p>This event has not program manager.</p>
-                      {% endif %}
-                    </div>
-                  <div class="col-md-6">
-                      <h3>Facilitators</h3>
-                              {% if eventFacilitatorNames %}
-                              <ul>
-                                    {% for facilitator in eventFacilitatorNames%}
-                                    <li>{{facilitator.firstName}} {{facilitator.lastName}}</li>
-                                    {% endfor %}
-                              </ul>
-                              {% else %}
-                                  <p>This event has no facilitators.</p>
-                              {% endif %}
-                    </div>
-                    <div>
-                        <div>
+                        <h3><br>Contact Information:</h3>
```

Comment:
We should handle the case where there isn't any contact info saved for the event. (probably just don't show the header)

Reply:
resolved